### DISCUSSION TITLE
通过邮件授权登录专业版无法收到验证码ld753@163.com

### DISCUSSION THREAD

**Discussion by ld753** [getlantern/lantern-forum-cn#390] [Q&A]
通过邮件授权登录专业版无法收到验证码ld753@163.com

**Comment by MaxMKMax** [comment]
@ld753 好的，您也可以给我您的付款收据吗？我将帮助您与团队联系并要求他们手动向您发送代码。请几天后查看您的电子邮件或 github。

给您带来的不便敬请谅解，感谢您对蓝灯的支持!

**Comment by MaxMKMax** [comment]
@ld753 您好. 请问您登陆进去了吗? 客服部门让您发邮件到 support@getlantern.org

**Comment by ld753** [comment]
太过分了！


--
发自我的网易邮箱手机智能版



在 2024-04-09 22:22:46，"Mike K" ***@***.***> 写道：


@ld753 您好. 请问您登陆进去了吗? 客服部门让您发邮件到 ***@***.***

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/ed152980-63db-41ca-a4fb-a7a632881e23)

@ld753 您好. 客服给您发邮件拿资料. 因为系统里完全没有您的购买记录. 要不您发购买单据给我吧.

您看以上的图显示您没有专业版记录. 是不是哪个环节出错了?

**Comment by ld753** [comment]
抱歉，我记错了我的注册邮箱，现在可以登陆了




--
发自我的网易邮箱手机智能版



在 2024-04-10 22:35:15，"Mike K" ***@***.***> 写道：


image.png (view on web)

@ld753 您好. 客服给您发邮件拿资料. 因为系统里完全没有您的购买记录. 要不您发购买单据给我吧.

您看以上的图显示您没有专业版记录. 是不是哪个环节出错了?

**Reply by MaxMKMax** [reply]
@ld753 好的. 没事. 能登陆就好. 谢谢您支持蓝灯!
